docs/master/federation/entity-representations.md

```diff
@@ -0,0 +1,85 @@
+# Entity representation
+
+In order for a subgraph to be able to return an entity from another subgraph, it is necessary to define in the first
+subgraph the representation of the required entity from those fields that are known to the first subgraph.
+Read more about the entity representation in the [Apollo Federation docs](https://www.apollographql.com/docs/federation/v1/entities/#entity-representations).
+
+A representation always consists of:
+- A `__typename` field;
+- Values for the entity's primary key fields.
+
+In this way, the resolver should return an array consisting of the `__typename` field, which corresponds to the entity's
+name, and the primary key that can identify the entity.
+
+### Example 1
+
+In this example, the `review-service` has an entity called `Review`, which in turn has an entity called `Product`
```

Comment:
What is the `review-service`? I think it would be clearer to simply put review service without the quotes.